app/api/routers/breeze_buddy/leads/handlers.py

```
@@ -107,21 +107,29 @@ async def push_lead_handler(req: PushLeadRequest, current_user: UserInfo) -> Dic
         f"User {current_user.username} (role: {current_user.role}) pushing lead "
-        f"for merchant: {req.merchant}, template: {req.template}"
+        f"for reseller: {reseller_id}, template: {req.template}"
     )
 
     try:
+
+        if not reseller_id or not req.identifier:
+            raise HTTPException(
+                status_code=status.HTTP_400_BAD_REQUEST,
+                detail="reseller (or merchant for backward compatibility) is required",
+            )
```

Ruff (0.15.1)

[warning] Abstract `raise` to an inner function

(TRY301)